client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+          }}
+          />
+        </LessonCover>
+        <p>
+          Deși la prima vedere pare un subiect simplu, sunt totuși multe de
+          știut despre link-uri și butoane în HTML. În acest articol vom discuta
+          despre markup, atribute și cele mai bune practici. Totodată vom trata
+          împreună cele două concepte pentru că de multe ori se folosesc
+          link-uri în loc de butoane și invers. Astfel la sfârșitul articolului
+          vom ști exact cum să implementăm un UX corect folosind cele 2
+          elemente.
+        </p>
+        <section>
+          <h2> The Basics </h2>
+          <LessonHeading as="h3" id="a-element">
+            {'<a> element'}
+          </LessonHeading>
+          <p>
+            Elementul
+            {' '}
+            <strong className="formatted">{'<a>'}</strong>
+            {' '}
+            denumit și
+            {' '}
+            <i>ancoră</i>
+            {' '}
+            sau
+            {' '}
+            <i>link</i>
+            {' '}
+            este unul din cele mai uzuale elemente
+            folosite pentru construirea paginilor web. Dacă vrem să ne mutam la
+            o altă pagină sau la o altă zonă în cadrul aceleiași pagini, atunci
+            vom folosi acest element. Modalitatea prin care adăugăm un link
+            într-o pagină este prin folosirea atributului
+            {' '}
+            <strong className="formatted">href</strong>
+            {' '}
+            care indică URL-ul
+            link-ului.
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="exemple-unde-folosim-tagul-a">
+            Exemple unde folosim tag-ul a
+          </LessonHeading>
+          <LessonHeading as="h4" id="cele-mai-uzuale">
+            Cele mai uzuale
+          </LessonHeading>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="https://frontend.ro/"> Frontend.ro </a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/link-to-a-page.png"
+            alt="Cel mai uzual mod de a folosi tag-ul a - URL-uri absolute"
+            demo="/demo/html/url-absolut"
+          />
+          <p>
+            În cazul acesta, spunem că este un link către un URL
+            {' '}
+            <strong>absolut</strong>
+            {' '}
+            - adică un URL care conține întreaga adresă
+            a unei pagini. Acest tip de URL este deseori întâlnit atunci când un
+            website face legătura către un alt website.
+          </p>
+          <p>
+            Putem avea și URL-uri
+            {' '}
+            <strong>relative</strong>
+            , de obicei folosite
+            la realizarea meniurilor paginilor web:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/team.html">Team</a>`}
```

Comment:
Relative links are not well explained. This > `/team.html` is not a relative link, but an absolute one. It will always start from the "root" and find the "team.html" page there.

To be a relative link it must lead us to a different destination, depending on our current location. 

This is a relative link `team.html` (without `/`). Also this `./team.html` or this -> `../team.html`.

For the demo, I think we should show a simple folder structure with 2 foldrse: `home` and `settings`. In `home` let's have a `home.html` and `logo.jpg`. Also in `settings` let's have a `settings.html` and a `logo.jpg`. If we add this link `<a href="logo.jpg"> open image in another tab </a>` to both of them, it will open up a different image.


PS: please read the full article again, and change all explanations related to relative links